yarn/pom.xml

```diff
@@ -164,6 +164,92 @@
      </dependency>
   </dependencies>
 
+  <profiles>
+    <!-- The history server uses the Hadoop 2.6+ REST API -->
```

Comment:
Can this say "The YARN application server..." there is already a different component in Spark called history server.